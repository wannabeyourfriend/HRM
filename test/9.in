2
11
inbox
copyto 0
inbox
copyto 1
copyfrom 0
sub 2
outbox
copyfrom 1
sub 0
outbox
jumpto 1