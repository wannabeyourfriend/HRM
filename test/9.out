Error on instruction 6